src/NUnitFramework/framework/Constraints/CollectionItemsEqualConstraint.cs

```diff
@@ -72,6 +72,20 @@ public CollectionItemsEqualConstraint Using(IComparer comparer)
             return this;
         }
 
+#if (NET_4_0 || NET_4_5) && !PORTABLE
```

Comment:
Well, if you include it and the portable build compiles, then it works. (It does, btw.) In fact in VS2015, you only have to type it and you get a popup showing which targets support it.

For testing, just run the tests for that particular build, which are in the individual runtime folders. For .NET 2.0, 4.0 and 4.5 use the console runner. For Silverlight use the test app itself. For Portable, you'll find a console app that links to the portable tests.